build_image.sh

```diff
@@ -87,11 +87,24 @@ generate_onie_installer_image()
           $ONIE_INSTALLER_PAYLOAD
 }
 
+generate_onie_device_list()
+{
+    # Generate asic-specific ONIE device list
+    rm -rf ./installer/x86_64/devices/
+    mkdir -p ./installer/x86_64/devices/
+    for d in `find -L ./device  -maxdepth 2 -mindepth 2 -type d`; do
+            if [ -f $d/platform_asic ] && grep -Fxq "$CONFIGURED_PLATFORM" $d/platform_asic; then
```

Comment:
>Fxq [](http://example.com/codeflow?start=48&length=3)

Will the pattern `broadcom` match the file content `broadcom-dnx`? I guess no, just double check.

Reply:
It will not match broadcom-dnx.